### PR TITLE
fix(file-picker): `pickFiles (...)` doesn't work first time on Safari

### LINKED ISSUE #14: bug(file-picker): `pickFiles (...)` doesn't work first time on Safari
**Plugin version:**
0.5.7

**Platform(s):**
Safari Web

**Current behavior:**
When first calling the command `pickFiles` on safari (ie from button click on first load of page) - nothing happens, it fails silently (no file picker shown). 
When calling the function a second time (pressing the button again), everything works as expected.

If you refresh the page, this same series of events occurs. so seems to maybe be an initialization related issue?

Everything is aok on other browsers so far!

Thanks!

**Expected behavior:**
First call to `pickFiles` should open the file picker

**Steps to reproduce:**
the quickest way to reproduce this is to run your capacitor plugin demo in safari and go to the FilePicker screen. 
Click the Pick Files button once - nothing happens
Click the Pick Files button a second time - file picker window opens

**Capacitor doctor:**

```
Latest Dependencies:

  @capacitor/cli: 4.6.3
  @capacitor/core: 4.6.3
  @capacitor/android: 4.6.3
  @capacitor/ios: 4.6.3

Installed Dependencies:

  @capacitor/cli: 4.6.1
  @capacitor/core: 4.6.1
  @capacitor/android: 4.6.1
  @capacitor/ios: 4.6.1

[success] iOS looking great! 👌
[success] Android looking great! 👌```

### PR DESCRIPTION
**Pull request by stephan-fischer**
## Pull request checklist

Please check if your PR fulfills the following requirements:

- [x] The changes have been tested successfully.
- [x] A changeset has been created (`npm run changeset`).
- [x] I have read and followed the [pull request guidelines](https://capawesome.io/contributing/pull-requests/).

Close #14

### CODE REVIEW

File: .changeset/funny-taxis-tie.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@capawesome/capacitor-file-picker': minor
+---
+
+bug(file-picker): pickFiles on Safari web does not bring up file picker on first call #14
```
Comment (robingenz) [inline .changeset/funny-taxis-tie.md]:
```suggestion
fix(web): `pickFiles (...)` doesn't work first time on Safari
```

File: .changeset/funny-taxis-tie.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+'@capawesome/capacitor-file-picker': minor
```
Comment (robingenz) [inline .changeset/funny-taxis-tie.md]:
```suggestion
'@capawesome/capacitor-file-picker': patch
```

File: packages/file-picker/src/index.ts [hunk 3]
```diff
@@ -1,9 +1,10 @@
 import { registerPlugin } from '@capacitor/core';
 
 import type { FilePickerPlugin } from './definitions';
+import * as web from './web';
 
 const FilePicker = registerPlugin<FilePickerPlugin>('FilePicker', {
-  web: () => import('./web').then(m => new m.FilePickerWeb()),
+    web:  () => new web.FilePickerWeb()
```
Comment (robingenz) [inline packages/file-picker/src/index.ts]:
```suggestion
  web:  () => new web.FilePickerWeb()
```

File: packages/file-picker/src/index.ts [hunk 4]
```diff
@@ -1,9 +1,10 @@
 import { registerPlugin } from '@capacitor/core';
 
 import type { FilePickerPlugin } from './definitions';
+import * as web from './web';
```
Comment (robingenz) [inline packages/file-picker/src/index.ts:5]:
```suggestion
// See https://github.com/capawesome-team/capacitor-plugins/issues/14
import * as web from './web';
```

**Review by robingenz — CHANGES_REQUESTED** [review]

**Review by stephan-fischer — COMMENTED** [review]
thank you 😊

**Review by stephan-fischer — COMMENTED** [review]
Reviewed

**Comment by stephan-fischer** [comment]
@robingenz could you test this code, or is it possible to test the code previously via github package etc? I think we are only one step closer - i dont know if it works on compiled ts code.

**Comment by robingenz** [comment]
@stephan-fischer Sure, i just published a dev version:

```
npm i @capawesome/capacitor-file-picker@5.1.1-dev.fbc4f0a.1703929046
```

**Comment by robingenz** [comment]
Before: https://capacitor-plugin-demo-8imy2i1j7-robingenz.vercel.app/
After: https://capacitor-plugin-demo-rcpboxsdr-robingenz.vercel.app/

**Comment by stephan-fischer** [comment]
> @stephan-fischer Sure, i just published a dev version:
> 
> ```
> npm i @capawesome/capacitor-file-picker@5.1.1-dev.fbc4f0a.1703929046
> ```

Thank you, it works 🤩

**Review by robingenz — APPROVED** [review]

**Pull request merged**